config/standard/resources/_id

```diff
@@ -0,0 +1 @@
+resources
```

Comment:
Let's prefix it with `medic-` to be consistent with our other ddocs. Also maybe it should be `medic-configuration` rather than `resources`?

Reply:
Ah, right, I see how you're doing it now. This is potentially problematic because people can change the icons on the `resources` doc and I think this will completely overwrite those configurations. So I think we'll need some additional code somewhere to merge the default images with the custom defined ones.